pkgs/by-name/hy/hybridbar/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  glib,
+  pango,
+  cairo,
+  gtk3,
+  gtk-layer-shell,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "HybridBar";
+  version = "0.4.9";
+
+  src = fetchFromGitHub {
+    owner = "vars1ty";
+    repo = "HybridBar";
+    rev = "v${version}";
```

```suggestion
    rev = "refs/tags/${version}";
```

There's no `v` prefix at all.